Description: Properly escape a dash in the django-admin manual page
 Fix a lintian I: message about improper usage of minus instead
 of hyphen.
 .
 This should be forwarded upstream.
Author: Raphael Hertzog <hertzog@debian.org>

--- a/docs/man/django-admin.1
+++ b/docs/man/django-admin.1
@@ -194,7 +194,7 @@ The domain of the message files (default
 .TP
 .I \-e, \-\-extension=EXTENSION
 The file extension(s) to examine (separate multiple
-extensions with commas, or use -e multiple times) (makemessages command).
+extensions with commas, or use \-e multiple times) (makemessages command).
 .TP
 .I \-s, \-\-symlinks
 Follows symlinks to directories when examining source code and templates for
